go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/celestiaorg/celestia-app
 
-go 1.21
-
-toolchain go1.21.0
```

Comment:
[question] I don't see anything about the `toolchain` directive being removed in Go 1.21.1 release notes so I'm wondering why this was removed. Did `go mod tidy` auto remove it?

Ref:
- https://tip.golang.org/doc/devel/release#go1.21.minor
- https://github.com/golang/go/issues?q=milestone%3AGo1.21.1+label%3ACherryPickApproved

Reply:
yes, it was removed automatically when I bumped the version